README.md

```diff
@@ -1229,14 +1279,14 @@ To define the `test-id` to a component use the follow structure: `[page-name||co
 
 **✅ Good:**
 
-- forgot-password__input--email
-- header__select--cnpjList
-- login__button--forgot-password
+- forgot-password\_\_input--email
```

Comment:
question: why this change?

Reply:
VSCode auto format fool me again :disappointed: 
I already revert it.

Reply:
Now the PR is empty @luiz-ortega 

![image](https://user-images.githubusercontent.com/75763403/216326244-020df258-aa39-48c8-9c74-9fc617a12458.png)

Reply:
@RodrigoRVSN yes, i had it reverted so i could readded all the alterations. It's solved now!